lib/assets/javascripts/chef/components/icon.js

```diff
@@ -0,0 +1,24 @@
+//= require ../chef
+
+(function($, window) {
+  'use strict';
+
+  Chef.Web.Core.components.icon = {
+    init: function() {
```

Comment:
Ah, that's true, I overlooked that. I'll add a note and think about how to make this better.